### ISSUE TITLE
I am getting an issue regarding memory, how to solve this ? 

### ISSUE THREAD

**Issue by devarajarun** [Merck/deepbgc#71]
(deepbgc) [devaraja@rocket ~]$ deepbgc pipeline deepbgc/deepbgc-0.1-3.29/test/data/BGC0000015.fa
 _____                  ____    ____   ____ 
 |  _ \  ___  ___ ____ | __ )  / ___) / ___)
 | | \ \/ _ \/ _ \  _ \|  _ \ | |  _ | |    
 | |_/ /  __/  __/ |_) | |_) || |_| || |___ 
 |____/ \___|\___| ___/|____/  \____| \____)
=================|_|===== version 0.1.29 =====
INFO    20/03 11:17:39   Loading model from: /gpfs/space/home/devaraja/.local/share/deepbgc/data/0.1.0/detector/deepbgc.pkl
Using TensorFlow backend.
WARNING 20/03 11:17:45   From /gpfs/space/home/devaraja/.conda/envs/deepbgc/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:74: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

WARNING 20/03 11:17:45   From /gpfs/space/home/devaraja/.conda/envs/deepbgc/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:517: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING 20/03 11:17:45   From /gpfs/space/home/devaraja/.conda/envs/deepbgc/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:4138: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

WARNING 20/03 11:17:45   From /gpfs/space/home/devaraja/.conda/envs/deepbgc/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:133: The name tf.placeholder_with_default is deprecated. Please use tf.compat.v1.placeholder_with_default instead.

WARNING 20/03 11:17:45   From /gpfs/space/home/devaraja/.conda/envs/deepbgc/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:3445: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
WARNING 20/03 11:17:46   From /gpfs/space/home/devaraja/.conda/envs/deepbgc/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:174: The name tf.get_default_session is deprecated. Please use tf.compat.v1.get_default_session instead.

terminate called after throwing an instance of 'std::system_error'
  what():  Resource temporarily unavailable
Aborted
